EU4toV2/Source/V2World/Output/outCountry.cpp

```diff
@@ -117,29 +127,52 @@ std::ostream& V2::operator<<(std::ostream& output, const Country& country)
 
 void V2::Country::outputCommons(std::ostream& output)
 {
-	output << "graphical_culture = UsGC\n";	// default to US graphics
+	output << "graphical_culture = ";
+	auto graphicalCulture = modCommons.getGraphicalCulture();
+	if (graphicalCulture.empty())
+		graphicalCulture = details.graphicalCulture;
+	if (!graphicalCulture.empty())
+		output << graphicalCulture << '\n';
+	else
+		output << "UsGC\n"; // default to US graphics
+	
 	if (nationalColors.getMapColor())
 		output << "color " << *nationalColors.getMapColor() << "\n";
-	for (const auto& party : details.parties) output << party;
+	else if (!modCommons.getColor().empty())
+		output << "color " << modCommons.getColor() << '\n';
```

Comment:
I made it a stringOfItem so it already includes `=`. And I decided on string type since it's just a copy of values in common/countries file that is not meant to be manipulated.

Short summary, countries coming from eu4 keep their colors, others get the HPM colors if the tag is defined there, else take from blankMod.